devices/StepperMotor/Threading_StepperMotor.py

```diff
@@ -0,0 +1,522 @@
+from PyQt5 import QtWidgets, QtCore, QtGui
+import sys, os
+from time import sleep
+import RPi.GPIO as GPIO
+
+from PyQt5.QtGui import *
```

Comment:
Should import only functions that are used within the file, import * isn't a great practice.